cargo-dist/src/init.rs

```diff
@@ -787,6 +787,10 @@ fn get_new_dist_metadata(
                 );
             }
         }
+    } else {
+        // homebrew doesn't exist in the new config
+        meta.tap = None;
+        publish_jobs.retain(|job| job != &PublishStyle::Homebrew);
```

Comment:
Good start, but this is a bit too aggressive.

The homebrew installer can be enabled per-package, but tap/publish settings can/should be global. So I think it's only sound to unset these settings if they specifically *unchecked* the homebrew installer in *this* run, because they're clearly trying to disable the homebrew installer globally. You can see a check like this in `homebrew_is_new` above (you just gotta invert it to make `homebrew_was_set`).

Reply:
I think I understood what to do, but I don't understand _why_. Thanks for the review!